kaluga-library-components/src/main/kotlin/Dependencies.kt

```diff
@@ -0,0 +1,190 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+import org.gradle.api.artifacts.dsl.DependencyHandler
```

Comment:
This unfortunately duplicates something we already want to migrate to that is part of standard Gradle now, [version catalogs](https://github.com/splendo/kaluga/issues/513).